src/main/java/io/github/genomicdatainfrastructure/discovery/datasets/infrastructure/ckan/mapper/CkanDatasetsMapper.java

```diff
@@ -58,12 +86,55 @@ public interface CkanDatasetsMapper {
     @Mapping(target = "count", ignore = true)
     ValueLabel map(CkanTag ckanTag);
 
-    @Mapping(target = "title", source = "name")
+    @Mapping(target = "accessUrl", source = "accessUrl")
+    @Mapping(target = "downloadUrl", source = "downloadUrl")
+    @Mapping(target = "description", source = "description")
+    @Mapping(target = "format", source = "format")
+    @Mapping(target = "languages", source = "language")
+    @Mapping(target = "retentionPeriod", source = "retentionPeriod")
+    @Mapping(target = "accessService", source = "accessServices")
```

Comment:
Not sure how mapstruct deals with these, but why are some mapped from plural to singular (languages -> language) while others are from singular to plural (accessService -> accessServices)?

Reply:
Not blocking for test.